### PR TITLE
Update tiflash-620-upgrade-guide.md

### PR DESCRIPTION
**Pull request by joey-yez**
add new statement for experiment feature: fastscan.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Only documents modified.

Add new statement for the experimental feature: FastScan.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@joey-yez @hfxsd Refined the structure to better fit the whole guide. PTAL.

File: tiflash-620-upgrade-guide.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ TiFlash v6.2.0 默认升级到 PageStorage V3 版本（对应配置项参数 [`f
...
 
-## 从 v6.1.0 升级至 v6.2.0
+### FastScan
 
-从 v6.1.0 升级至 v6.2.0 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [PageStorage](#pagestorage)。
+该功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关功能的形态和使用方式有可能在后续版本中发生变化。
+
+## 从 v6.1 升级至 v6.2
+
+从 v6.1 升级至 v6.2 时，需要注意 v6.2.0 PageStorage 变更数据版本和引入实验特性 FastScan 带来的影响。具体请参考 [PageStorage](#pagestorage) 和 [FastScan](#fastscan)。
```
Comment (moyun) [inline tiflash-620-upgrade-guide.md:88]: 这个说法按下面改一下，这里只是需要提醒用户实验功能是可能发生形态改变的，不必再提 “PageStorage 变更数据版本”了，这个不是实验功能。 参考一下这个：[PR](https://github.com/pingcap/docs-cn/pull/10831)

> ### 关于 v6.2.0 的新增实验功能
> 
> v6.2.0 新增了一项名为[fastScan｜fastmode](/tiflash/？？？) 的实验功能。该实验功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关实验功能的形态和使用方式有可能在后续版本中发生变化，提请用户在使用中注意。

**Review by moyun — CHANGES_REQUESTED** [review]
please update accordingly

File: tiflash-620-upgrade-guide.md [hunk 2]
```diff
@@ -44,22 +46,22 @@ TiFlash 在 v6.1.0 对 Proxy 做了升级（与 TiKV v6.0.0 对齐）。新的 P
...
-- TiDB v6.0.0 及之前版本：默认关闭动态分区裁剪。旧版本升级遵循已有设定，不会自动开启（相对的也不会关闭）此功能。
+- TiDB v6.0 及之前版本：默认关闭动态分区裁剪。旧版本升级遵循已有设定，不会自动开启（相对的也不会关闭）此功能。
 
     升级完成之后，如果要启用动态分区裁剪特性，请确保 `tidb_partition_prune_mode` 的值为 `dynamic`，并手动更新分区表的全局统计信息。关于如何手动更新统计信息，参见[动态裁剪模式](/partitioned-table.md#动态裁剪模式)。
 
-## 从 v5.x.x 或 v6.0.0 升级至 v6.2.0
+## 从 v5.x 或 v6.0 升级至 v6.2
 
-TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x.x 或 v6.0.0 升级至 v6.2.0 时，除了需要注意 [TiFlash Proxy](#tiflash-proxy) 和[动态分区裁剪](#动态分区裁剪)的变化，还应注意 PageStorage 功能的变化。
+TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，同时，引入了一个新的实验特性 [FastScan](/develop/dev-guide-use-fastscan.md)，因此，从 v5.x 或 v6.0 升级至 v6.2 时，除了需要注意 [TiFlash Proxy](#tiflash-proxy) 和[动态分区裁剪](#动态分区裁剪)的变化，还应注意 PageStorage 和 FastScan 功能引入带来的注意事项。
```
Comment (shichun-0415) [inline tiflash-620-upgrade-guide.md:57]:
```suggestion
TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6.0 升级至 v6.2 时，除了需要注意 [TiFlash Proxy](#tiflash-proxy) 和[动态分区裁剪](#动态分区裁剪)的变化，还应注意 PageStorage 和 FastScan 功能的变化。
```

File: tiflash-620-upgrade-guide.md [hunk 3]
```diff
@@ -79,6 +81,10 @@ TiFlash v6.2.0 默认升级到 PageStorage V3 版本（对应配置项参数 [`f
 
 强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[缩容 TiFlash 节点](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
 
-## 从 v6.1.0 升级至 v6.2.0
+### FastScan
+
+该功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关功能的形态和使用方式有可能在后续版本中发生变化。
+
```
Comment (shichun-0415) [inline tiflash-620-upgrade-guide.md:87]:
```suggestion
```

File: tiflash-620-upgrade-guide.md [hunk 4]
```diff
@@ -79,6 +81,10 @@ TiFlash v6.2.0 默认升级到 PageStorage V3 版本（对应配置项参数 [`f
...
+## 从 v6.1 升级至 v6.2
 
-从 v6.1.0 升级至 v6.2.0 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [PageStorage](#pagestorage)。
+从 v6.1 升级至 v6.2 时，需要注意 v6.2.0 PageStorage 变更数据版本和引入实验特性 FastScan 带来的影响。具体请参考 [PageStorage](#pagestorage) 和 [FastScan](#fastscan)。
```
Comment (shichun-0415) [inline tiflash-620-upgrade-guide.md:90]:
```suggestion
从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [PageStorage](#pagestorage)。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash-620-upgrade-guide.md [hunk 5]
```diff
@@ -44,22 +46,22 @@ TiFlash 在 v6.1.0 对 Proxy 做了升级（与 TiKV v6.0.0 对齐）。新的 P
...
 
-- TiDB v6.0.0 及之前版本：默认关闭动态分区裁剪。旧版本升级遵循已有设定，不会自动开启（相对的也不会关闭）此功能。
+- TiDB v6.0 及之前版本：默认关闭动态分区裁剪。旧版本升级遵循已有设定，不会自动开启（相对的也不会关闭）此功能。
 
     升级完成之后，如果要启用动态分区裁剪特性，请确保 `tidb_partition_prune_mode` 的值为 `dynamic`，并手动更新分区表的全局统计信息。关于如何手动更新统计信息，参见[动态裁剪模式](/partitioned-table.md#动态裁剪模式)。
 
-## 从 v5.x.x 或 v6.0.0 升级至 v6.2.0
+## 从 v5.x 或 v6.0 升级至 v6.2
 
-TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x.x 或 v6.0.0 升级至 v6.2.0 时，除了需要注意 [TiFlash Proxy](#tiflash-proxy) 和[动态分区裁剪](#动态分区裁剪)的变化，还应注意 PageStorage 功能的变化。
```
Comment (moyun) [inline tiflash-620-upgrade-guide.md:55]: 这句话精简一下吧：
TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此从 v5.x 或 v6.0 升级至 v6.2 时，除了需要注意 [TiFlash Proxy](https://github.com/pingcap/docs-cn/blob/79b423973f32f4e7e2c26df77e41a253428a1720/tiflash-620-upgrade-guide.md#tiflash-proxy) 和[动态分区裁剪](https://github.com/pingcap/docs-cn/blob/79b423973f32f4e7e2c26df77e41a253428a1720/tiflash-620-upgrade-guide.md#%E5%8A%A8%E6%80%81%E5%88%86%E5%8C%BA%E8%A3%81%E5%89%AA)的变化，还应注意 以下几点注意事项。

File: tiflash-620-upgrade-guide.md [hunk 6]
```diff
@@ -79,6 +77,6 @@ TiFlash v6.2.0 默认升级到 PageStorage V3 版本（对应配置项参数 [`f
 
 强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[缩容 TiFlash 节点](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
 
-## 从 v6.1.0 升级至 v6.2.0
+## 从 v6.1 升级至 v6.2
 
-从 v6.1.0 升级至 v6.2.0 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [PageStorage](#pagestorage)。
+从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [PageStorage](#pagestorage)。
```
Comment (shichun-0415) [inline tiflash-620-upgrade-guide.md:82]:
```suggestion
从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考 [从 v5.x 或 v6.0 升级至 v6.2](#从-v5x-或-v60-升级至-v62) 中关于 PageStorage 的描述。
```

**Review by moyun — COMMENTED** [review]
LGTM

**Review by moyun — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**